packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/AddressInfoDropdown.tsx

```diff
@@ -45,7 +45,10 @@ export const AddressInfoDropdown = ({
   return (
     <>
       <details ref={dropdownRef} className="dropdown dropdown-end leading-3">
-        <summary tabIndex={0} className="btn btn-secondary btn-sm pl-0 pr-2 shadow-md dropdown-toggle gap-0 !h-auto">
+        <summary
+          tabIndex={0}
+          className="btn btn-secondary uppercase btn-sm pl-0 pr-2 shadow-md dropdown-toggle gap-0 !h-auto"
```

Comment:
Now that I see the `uppercase` here a couple of things:

1. Should we remove the uppercase here? It was how it was displayed before... but it might make more sense to have it normal: 0x VS. 0X and also the checksum stuff (0x40…16a9 VS. 0X40…16A9). Even for ENS seems more natural. Just my opinion!
2. Since buttons are not uppercased by default, let's remove all the `normal-case` classes

Reply:
> but it might make more sense to have it normal: 0x VS. 0X and also the checksum stuff (0x40…16a9 VS. 0X40…16A9). Even for ENS seems more natural. Just my opinion!

Yup makes sense, makes it visually consistent too with other wallets like MM !! updated at [81c22cc](https://github.com/scaffold-eth/scaffold-eth-2/pull/656/commits/81c22cc8d37c78d5d08e10c40bea73d540227962) 

>Since buttons are not uppercased by default, let's remove all the normal-case classes
Ohh yes,  tysm updated at [f5b2e57](https://github.com/scaffold-eth/scaffold-eth-2/pull/656/commits/f5b2e57a581e46634ccaa9bf6c4eee08765e798c)